taxii2-threat-intel-feed/release_notes.md

```diff
@@ -1 +1,7 @@
-#### What's Improved
\ No newline at end of file
+#### What's Improved
+
+- Added a new config parameter 'Headers'.
+- Added a new parameter "Headers" in following actions:
```

Comment:
Also add new action added `Download Indicators`, note that this action is added for FCP no need to add in  playbooks